features/src/main/feature/camel-features.xml

```diff
@@ -457,21 +457,19 @@
         <bundle dependency='true'>wrap:mvn:software.amazon.awssdk/translate/${aws-java-sdk2-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-aws2-translate/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-aws-secrets-manager' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk2/${aws-sdk-v2-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-xml/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-aws-secrets-manager/${project.version}</bundle>-->
-<!--    </feature>-->
 <!--    <feature name='camel-aws-cloudtrail' version='${project.version}' start-level='50'>-->
 <!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
 <!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aws-java-sdk2/${aws-sdk-v2-bundle-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-aws-cloudtrail/${project.version}</bundle>-->
 <!--    </feature>-->
+    <feature name='camel-aws-secrets-manager' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-cloudevents</feature>
```

Comment:
Used camel-cloudevents as feature instead of a single bundle

Reply:
I don't see `camel-cloudevents` in the dependency tree:
```
[INFO] -------------< org.apache.camel:camel-aws-secrets-manager >-------------
[INFO] Building Camel :: AWS2 Secrets Manager 4.5.0
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- dependency:3.6.1:tree (default-cli) @ camel-aws-secrets-manager ---
[INFO] org.apache.camel:camel-aws-secrets-manager:jar:4.5.0
[INFO] +- org.apache.camel:camel-support:jar:4.5.0:compile
[INFO] |  +- org.apache.camel:camel-api:jar:4.5.0:compile
[INFO] |  |  \- jakarta.xml.bind:jakarta.xml.bind-api:jar:4.0.2:compile
[INFO] |  |     \- jakarta.activation:jakarta.activation-api:jar:2.1.3:compile
[INFO] |  +- org.apache.camel:camel-management-api:jar:4.5.0:compile
[INFO] |  +- org.apache.camel:camel-util:jar:4.5.0:compile
[INFO] |  +- org.apache.camel:camel-xml-jaxp-util:jar:4.5.0:compile
[INFO] |  \- org.slf4j:slf4j-api:jar:2.0.12:compile
[INFO] +- org.apache.camel:camel-util-json:jar:4.5.0:compile
[INFO] +- software.amazon.awssdk:secretsmanager:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:aws-json-protocol:jar:2.25.15:compile
[INFO] |  |  \- software.amazon.awssdk:third-party-jackson-core:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:protocol-core:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:http-auth-aws:jar:2.25.15:compile
[INFO] |  |  +- software.amazon.awssdk:checksums-spi:jar:2.25.15:compile
[INFO] |  |  \- software.amazon.awssdk:checksums:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:sdk-core:jar:2.25.15:compile
[INFO] |  |  +- software.amazon.awssdk:profiles:jar:2.25.15:compile
[INFO] |  |  \- org.reactivestreams:reactive-streams:jar:1.0.4:compile
[INFO] |  +- software.amazon.awssdk:auth:jar:2.25.15:compile
[INFO] |  |  \- software.amazon.eventstream:eventstream:jar:1.0.1:compile
[INFO] |  +- software.amazon.awssdk:http-auth-spi:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:http-auth:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:identity-spi:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:http-client-spi:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:regions:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:annotations:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:utils:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:aws-core:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:metrics-spi:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:json-utils:jar:2.25.15:compile
[INFO] |  +- software.amazon.awssdk:endpoints-spi:jar:2.25.15:compile
[INFO] |  \- software.amazon.awssdk:netty-nio-client:jar:2.25.15:runtime
[INFO] |     +- io.netty:netty-codec-http:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-codec-http2:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-codec:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-transport:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-common:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-buffer:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-handler:jar:4.1.107.Final:runtime
[INFO] |     |  \- io.netty:netty-transport-native-unix-common:jar:4.1.107.Final:runtime
[INFO] |     +- io.netty:netty-transport-classes-epoll:jar:4.1.107.Final:runtime
[INFO] |     \- io.netty:netty-resolver:jar:4.1.107.Final:runtime
[INFO] +- software.amazon.awssdk:cloudtrail:jar:2.25.15:compile
[INFO] +- software.amazon.awssdk:apache-client:jar:2.25.15:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.13:compile
[INFO] |  |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] |  +- org.apache.httpcomponents:httpcore:jar:4.4.13:compile
[INFO] |  \- commons-codec:commons-codec:jar:1.16.1:compile
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.16.2:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.16.2:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.16.2:compile
[INFO] \- org.apache.camel:camel-health:jar:4.5.0:compile
[INFO]    \- org.apache.camel:camel-core-engine:jar:4.5.0:compile
[INFO]       +- org.apache.camel:camel-base-engine:jar:4.5.0:compile
[INFO]       |  \- org.apache.camel:camel-base:jar:4.5.0:compile
[INFO]       \- org.apache.camel:camel-core-reifier:jar:4.5.0:compile
[INFO]          +- org.apache.camel:camel-core-model:jar:4.5.0:compile
[INFO]          \- org.apache.camel:camel-core-processor:jar:4.5.0:compile
[INFO] ------------------------------------------------------------------------
```